beacon_node/beacon_chain/src/migrate.rs

```diff
@@ -575,10 +545,9 @@ impl<E: EthSpec, Hot: ItemStore<E>, Cold: ItemStore<E>> BackgroundMigrator<E, Ho
                             // checkpoint, i.e. there aren't any forks starting at a block that is a
                             // strict ancestor of old_finalized_checkpoint.
                             warn!(
-                                log,
-                                "Found a chain that should already have been pruned";
-                                "head_block_root" => format!("{:?}", head_hash),
-                                "head_slot" => head_slot,
+                                head_block_root = %head_hash,
```

Comment:
I actually think we want to keep the `Debug` impl here since I believe the `Display` impl truncates the centre.

```suggestion
                                head_block_root = ?head_hash,
```